backend/src/schema/resolvers/users/location.spec.js

```diff
@@ -19,48 +19,48 @@ const updateUserMutation = gql`
 const newlyCreatedNodesWithLocales = [
   {
     city: {
-      lng: -74.5763,
-      lat: 41.1534,
+      id: expect.stringContaining('place'),
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-03-24T12:38:29Z" title="Tuesday, March 24th 2020, 1:38:29 pm +01:00">Mar 24, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -19,48 +19,48 @@ const updateUserMutation = gql`
 const newlyCreatedNodesWithLocales = [
   {
     city: {
-      lng: -74.5763,
-      lat: 41.1534,
+      id: expect.stringContaining('place'),
```


Why do these ids change? Do we make API calls during testing?